sensors/mediation/file_contexts

```diff
@@ -0,0 +1 @@
+/(vendor|system/vendor)/bin/hw/android\.hardware\.sensors\.intel@1\.0-service u:object_r:hal_sensors_default_exec:s0
```

Comment:
If google provided service is not going to be modified, please use google provided service

Reply:
Updated sensor service name to google default